README.md

```diff
@@ -103,6 +103,15 @@ docker run -d --name open360-api --restart=always -p 9000:9000 \
     -e GOOGLE_REDIRECT_URL \
     -e GOOGLE_CLIENT_ID \
     -e GOOGLE_CLIENT_SECRET \
+    -e FACEBOOK_REDIRECT_URL \
+    -e FACEBOOK_CLIENT_ID \
+    -e FACEBOOK_CLIENT_SECRET \
+    -e VK_REDIRECT_URL \
+    -e VK_CLIENT_ID \
+    -e VK_CLIENT_SECRET \
+    -e TWITTER_CALLBACK_URL \
```

Comment:
Nit: since Twitter integration is in "experimental" mode, possibly it would be better to hide this parameters or, at least, note that workability is not guaranteed.

Reply:
And I mean just a note, not a full-fledged documentation

Reply:
Makes sense, let me hide these parameters from the documentation. I think it won't harm anyone to leave those in the startup script, since they will not be passes if not set anyways. If you set them, then you probably know what you're doing